libraries/botbuilder-core/src/activityHandler.ts

```diff
@@ -633,6 +666,48 @@ export class ActivityHandler extends ActivityHandlerBase {
         await this.handle(context, 'UnrecognizedActivityType', this.defaultNextEvent(context));
     }
 
+    private getAdaptiveCardInvokeValue(activity: Activity): AdaptiveCardInvokeValue {
+        const { value } = activity;
+        if (!value) {
+            throw this.createAdaptiveCardInvokeException(
```

Comment:
This is throwing a method that [throws](https://github.com/microsoft/botbuilder-js/pull/3327/files#diff-73672579d87cdfbc0d4fe836801679f0e66c122783b3b28781a95ded45580882R699).

Reply:
Yes it is! My mistake.

Reply:
Fixed based on https://github.com/microsoft/botbuilder-dotnet/pull/5227/files#diff-7a382816f3c128c2daeb75ea350f34c5410ff57febaa968ada7e17dabb039224R707